### PR TITLE
add optional parameter to buy product

### PR DESCRIPTION
**Pull request by hcvcastro**
The integrator buy product page for new customers

Signed-off-by: Henry Castro <hcastro@collabora.com>

### CODE REVIEW

**Comment by juliushaertl** [comment]
@hcvcastro Maybe you can reference the upstream implementation once merged, as @gokaysatir mentioned that this is still ongoing.

**Comment by juliushaertl** [comment]
@hcvcastro @gokaysatir Do you have any update here? Is this still relevant?

**Comment by gokaysatir** [comment]
@juliushaertl hello :) Yes that's still relevant. It's tested on our side. Most of the new improvements are merged but we'll have 2 more before making this PR useful.

You can merge this PR if you wish. It is not supposed to effect anything for now.

**Review by juliushaertl — APPROVED** [review]

**Pull request merged**
